guides/advanced_guides/filtering.md

```diff
@@ -114,6 +114,12 @@ Say you want to only show to your user movies that were released after a certain
 release_date > 795484800 // march 18, 1995
 ```
 
+Querying on `Avengers`, the above example results in the following URL:
+
+```curl
+/indexes/movies/search?q=Avengers&filters=release_date%20%3E%20795484800
+```
+
 Now imagine that we want only the movies released after the 18 of march 1995, and directed by either Jordan Peel or Tim Burton, then you would use this filter:
```

Comment:
@bidoubiwa @eskombro I added an example URL to the example section, tell me what you think... I am not completely sure.

Reply:
I think that's a good idea! It's a real-life example

Reply:
cool, let's do it